start-here.mdx

```diff
@@ -1,8 +1,153 @@
 ---
 title: "Start here"
-icon: rocket
 ---
 
-import { Stub } from '/snippets/stub.jsx';
+import { Aside } from "/snippets/aside.jsx";
 
-<Stub issue="79" />
+This documentation is divided into several sections, with lower-level details appearing later in the list. For example, an existing standard contract may already solve the problem, so there may be no need to implement a new one.
+
+|                        |                                                                                              |
+| ---------------------- | -------------------------------------------------------------------------------------------- |
+| Ecosystem              | Tools and libraries for developing on TON, and how to use them.                              |
+| Payment processing     | Implementation guides for an exchange.                                                       |
+| Standard contracts     | Working with most popular standardized contracts.                                            |
+| Contract development   | Guides on developing smart-contracts.                                                        |
+| Languages              | Reference documentation for TON-specific programming languages.                              |
+| TON Virtual Machine    | Description of the low-level language that runs smart-contracts, and details of the runtime. |
+| Blockchain foundations | Comprehensive description of the blockchain.                                                 |
+| Contribute             | Documentation on writing this documentation.                                                 |
+
+## The minimum
```

Comment:
do we need a sub-header here?

<img width="301" height="102" alt="image" src="https://github.com/user-attachments/assets/4f770a79-3b53-4e75-a93f-38e758aed66a" />